<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>801</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="cesar">
     <attribute name="title">
      <string>Busqueda por KWIC</string>
     </attribute>
     <widget class="QPushButton" name="boton_buscar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dato_buscador">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Key Word In Context &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="texto1">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>811</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/cct/logo UCB.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>401</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QUndoView" name="undoView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>291</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>735</x>
        <y>270</y>
        <width>381</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <zorder>boton_buscar</zorder>
     <zorder>dato_buscador</zorder>
     <zorder>label_4</zorder>
     <zorder>texto1</zorder>
     <zorder>listView</zorder>
     <zorder>undoView</zorder>
     <zorder>listWidget</zorder>
     <zorder>label_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="C:/Users/Alejo/Pictures/logo/logo3.qrc"/>
 </resources>
 <connections/>
</ui>
